code/controllers/subsystem/fire_burning.dm

```diff
@@ -37,3 +37,41 @@ SUBSYSTEM_DEF(fire_burning)
 		if (MC_TICK_CHECK)
 			return
 
+SUBSYSTEM_DEF(cold_fire_burning)
```

Comment:
this doesnt need a second subsystem

Reply:
i'll remove it if not needed

Reply:
It's doing essentialy the same thing as the fire subsystem, if anything needs different behavior (different fire types) I'd rather have that datumized in some form.